consensus/consensus.md

```diff
@@ -277,6 +283,8 @@ The gas of a transaction is the sum of:
 | Channel offchain       |            0   |     0 |
 | Contract create        | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(ContractCreateTx) * GasPerByte`. It also includes gas for contract execution. |
 | Contract call          | `30 * BaseGas` | Proportional to the byte size of the transaction, specifically: `byte_size(ContractCallTx) * GasPerByte`. It also includes gas for contract execution. |
+| GA Attach              | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(GAAttachTx) * GasPerByte`. It also includes gas for execution of the init function. |
+| GA Meta                | `5 * BaseGas`  | Proportional to the byte size of the transaction, specifically: `byte_size(GAMetaTx) * GasPerByte`. It also includes gas for execution of the authentication function + gas for the wrapped transaction. |
```

Comment:
Considering [`aetx:gas_limit`](https://github.com/aeternity/aeternity/blob/be71889c61c107cd848e97d3c58623ca0e93581b/apps/aetx/src/aetx.erl#L257-L258) and `aega_meta_tx:gas_limit`, I do not see how the base gas of the inner tx is not accounted for this gas (i.e. equivalent gas of tx for micro block gas limit).

Reply:
"gas for the wrapped transactions" is not very precise, it would and should include also base gas...